### PR TITLE
[TASK] Overhaul common pages indentation

### PR DESCRIPTION
**Pull request by linawolf**
releases: main, 12.4, 11.5

### CODE REVIEW

File: Documentation/Introduction/Index.rst [hunk 1]
```diff
@@ -111,25 +119,29 @@ Each table is further defined by an array which configures how the
...
-       ],
-       'palettes' => [
-           ....
-       ],
-   ];
+The general structure (looking at a single table) is as follows:
+
+
+..  code-block::php
+    :caption: EXT:my_extension/Configuration/Overrides/tx_examples_haiku.php
```
Comment (brotkrueml) [inline Documentation/Introduction/Index.rst:126]: I am not sure if "Overrides" is the correct path here: In the example you override the whole TCA for tx_examples_haiku - which is not recommended in Overrides, I assume. This is more a "how is the TCA built", not "how you should override and place it", so I suggest to remove the caption altogether.

Applies also above for "Table entries (first level)"

Comment (linawolf) [reply Documentation/Introduction/Index.rst:126]: Mhh, the syntax would not be correct directly in the TCA folder either. What would you recommend as a caption to make this clear?

Comment (linawolf) [reply Documentation/Introduction/Index.rst]: Like this maybe?

Comment (brotkrueml) [reply Documentation/Introduction/Index.rst]: Does the caption "The array looks like this" really helps? I would remove the caption, we don't need it IMHO.

Comment (linawolf) [reply Documentation/Introduction/Index.rst]: Then you cannot know if the information where to put this is only missing or if this not an example you should use

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**